Seeder/source/models.py

```diff
@@ -407,6 +408,14 @@ def legacy_screenshot(self):
         """
         return
 
+    @property
+    def screenshot_file_exists(self):
+        try:
+            self.screenshot.file
```

Comment:
eh, tady by podle me stacilo neco jako bool(self.screenshot)

Reply:
To by ti vratilo, jestli ten objekt ma nastaveny nejaky screenshot nebo ne, ale pokud ten screenshot ma a ja jenom nemam ten soubor, tak to stejne vrati `True` a mne se nenacte stranka. V produkci, kde ty screenshoty jsou, by to nemelo zmenit v podstate nic, tohle mam spis aby mi to neshazovala polovina zdroju lokalne, kdyz je testuju.

Reply:
mhmm, tak stejne by bylo lepsi neco jako self.screeenshot.file.exists()

Reply:
Souhlas, ale:
![image](https://user-images.githubusercontent.com/1846915/60431240-962ce080-9bff-11e9-96bf-25d7cc9002a6.png)
Pokud soubor neexistuje, tak to uz na `self.screenshot.file` vyhodi exception. Jako asi by to slo treba nejak pres `path.exists()`, ale tbh tohle je jenom o trochu delsi and it does the trick